examples/page_post_comments_edge.js

```diff
@@ -34,8 +34,8 @@ fields = [
 ];
 params = {
 };
-const commentss = (new PagePost(id)).getComments(
+const comments = (new PagePost(id)).getComments(
   fields,
   params
 );
-logApiCallResult('commentss api call complete.', commentss);
\ No newline at end of file
+logApiCallResult('comments api call complete.', comments);
```

Comment:
Just a minor feedback, this change removed the newline which was part of the code earloer.